x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_summary_widget_full_size.stories.tsx

```diff
@@ -23,3 +23,14 @@ export const FullSize = {
     },
   },
 };
+
+export const FullSizeWithoutChart = {
+  args: {
+    ...mockedAlertSummaryResponse,
+    shouldHideCharts: true,
```

Comment:
Would you please add this prop to `FullSize` story? I think it would be better to have all the props available in `FullSize` story and not have a separate story for each prop :)

Reply:
I set it to false by default here so it's not confusing that the chart is missing and it's available to control so it can be switched in the main story.

Reply:
Sounds great 👍🏻